### PR TITLE
LPS-142377, LPS-142378, LPS-142417, LPS-142508, LPS-143204, LPS-143222 | master

### PR DESCRIPTION
**Pull request by Tim-Cao**
https://issues.liferay.com/browse/LPS-142377
https://issues.liferay.com/browse/LPS-142378
https://issues.liferay.com/browse/LPS-142417
https://issues.liferay.com/browse/LPS-142508
https://issues.liferay.com/browse/LPS-143204
https://issues.liferay.com/browse/LPS-143222

This is only for master.

@lfbesada
@sandrodw3

### CODE REVIEW

**Comment by Tim-Cao** [comment]
ci:test:sf

**Comment by Tim-Cao** [comment]
ci:test:relevant

**Comment by Tim-Cao** [comment]
ci:test:echo

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js [hunk 1]
```diff
@@ -105,66 +118,87 @@ export function MenuItem({item}) {
...
+								<ClayLayout.ContentCol gutters>
+									<ClayButtonWithIcon
+										aria-label={Liferay.Util.sub(
+											Liferay.Language.get('delete-x'),
+											`${title} (${type})`
+										)}
+										displayType="unstyled"
+										onClick={() =>
+											categoriesMultipleSelectionEnabled &&
+											!!item.children.length
```
Comment (p2kmgcl) [inline modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js:203]: `!!` is not needed here (and if needed, we prefer using `Boolean(val)`) :smile:

**Review by p2kmgcl — COMMENTED** [review]
PR looks nice, I just added small comment :partying_face:

File: modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js (hunk 1 above)
Comment (sandrodw3) [reply modules/apps/site-navigation/site-navigation-admin-web/src/main/resources/META-INF/resources/site_navigation_menu_editor/components/MenuItem.js:203]: ❤️

**Comment by sandrodw3** [comment]
I sent https://github.com/liferay-echo/liferay-portal/pull/6574 with the frontend changes applied 😄

**Pull request closed without merging**